akka-projection-r2dbc/src/main/scala/akka/projection/r2dbc/R2dbcProjectionSettings.scala

```diff
@@ -84,14 +102,19 @@ final class R2dbcProjectionSettings private (
     val useConnectionFactory: String,
     val timeWindow: JDuration,
     val backtrackingWindow: JDuration,
+    val deleteAfter: JDuration,
+    @deprecated("Not used, evict is only based on time window", "1.6.3")
```

Comment:
There are different deprecated since versions in this PR. Latest projections version is 1.6.5, so these should all be updated to 1.6.6.

```suggestion
    @deprecated("Not used, evict is only based on time window", "1.6.6")
```